### ISSUE TITLE
Should /mingw64/bin/libreadline7.dll be /mingw64/bin/libreadline.dll

### ISSUE THREAD

**Issue by oroppas** [Alexpux/MINGW-packages#2106]
I'm building Python-3.6.0 (I managed to rebase all the patches)
During the build process, I got

``ldd: C:/msys64/mingw64/lib/libreadline.dll.a: Exec format error``

Python build process is unable to link ``libreadline.dll`` since mingw package readline library
as ``libreadline7.dll``

```
mingw-w64-x86_64-readline /mingw64/bin/libreadline7.dll
mingw-w64-x86_64-readline /mingw64/include/
mingw-w64-x86_64-readline /mingw64/include/readline/
mingw-w64-x86_64-readline /mingw64/include/readline/chardefs.h
mingw-w64-x86_64-readline /mingw64/include/readline/history.h
mingw-w64-x86_64-readline /mingw64/include/readline/keymaps.h
mingw-w64-x86_64-readline /mingw64/include/readline/readline.h
mingw-w64-x86_64-readline /mingw64/include/readline/rlconf.h
mingw-w64-x86_64-readline /mingw64/include/readline/rlstdc.h
mingw-w64-x86_64-readline /mingw64/include/readline/rltypedefs.h
mingw-w64-x86_64-readline /mingw64/include/readline/tilde.h
mingw-w64-x86_64-readline /mingw64/lib/
mingw-w64-x86_64-readline /mingw64/lib/libhistory.a
mingw-w64-x86_64-readline /mingw64/lib/libhistory.dll.a
mingw-w64-x86_64-readline /mingw64/lib/libreadline.a
mingw-w64-x86_64-readline /mingw64/lib/libreadline.dll.a
```

Should it be re-named? If so, I'll make a pull request.

**Comment by Alexpux** [comment]
No libreadline is now have "7" version. It is ok.

**Comment by lazka** [comment]
I just hit the same thing, but I think it can be ignored. The build fails right after the error message due to a broken os.spawnv when trying to compile things. Looks like spawnv is broken when HAVE_WSPAWNV in posixmodule.c is not defined. Just defining it fixes everything.

After I clean things up I'll create a PR

**Comment by lazka** [comment]
I've opened #2587 

Feedback welcome!

**Comment by lazka** [comment]
See https://github.com/Alexpux/MINGW-packages/blob/975704607739bab3ce3cf782e65190880609d626/mingw-w64-python3/1610-fix-have-wspawnv.patch

should be fixed since the update to 3.6.x
